### PR TITLE
chore(pipeline): cleanup after branch rename

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

File: .releaserc.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module.exports = {
-    branches: ["main"],
+    branches: [
+        {name: 'v1', range: '0.x.x', channel: '0.x.x'},
```
Comment (fforootd) [inline .releaserc.js:3]: does this make sense? range 0?

Comment (livio-a) [reply .releaserc.js:3]: as we've never release a 1.0 version the current is still 0.3.5

Comment (fforootd) [reply .releaserc.js:3]: ok but in that case would the branch no be v1?

**Comment by muhlemmer** [comment]
ping @livio-a , @fforootd 
Should we move ahead with this PR?

**Comment by fforootd** [comment]
I think it is easier to recreate this anew 😁

**Pull request closed without merging**